astropy/coordinates/representation.py

```diff
@@ -2574,8 +2574,7 @@ def from_representation(cls, representation, base=None):
             d_lon_coslat = cls._get_d_lon_coslat(representation.d_phi, base)
             return cls(d_lon_coslat, -representation.d_theta)
 
-        return super(UnitSphericalDifferential,
```

Comment:
I *think* this is a behavior change, because this is happening in `UnitSphericalCosLatDifferential`, so its super is *not* the same as `UnitSphericalDifferential` ...

I think this change is probably *right*, and the original code was a copy-and-paste error, but @adrn should confirm if possible.

Reply:
That was my code, and, yes, definitely a copy&paste error (of course, the super of both is the same class, so it never mattered).

Reply:
Yea, seems fine